[
    {
        "category": "CONFIG",
        "classType": "JOB",
        "activityInitiatedType": "MANUAL",
        "result.status": "FAILED",
        "result.error.code": "ASDS0030",
        "result.error.detailedDescription": "Configuration of this protection policy cannot be completed because an error occurred on the application agent host.",
        "result.error.extendedReason": null,
        "result.error.reason": "Unable to configure the policy because of an error on the application agent.",
        "result.error.remediation": "To resolve this issue:\n1. Delete the files in the following directories:\nOn Windows, \\INSTALL_DIR\\AGENTSERVICECONFIG.\nOn Linux and AIX, /opt/dpsapps/agentsvc/config.\n2. In the PowerProtect Data Manager UI, go to Protection > Protection Policies, select this policy, and click Set Lockbox.\nIf the issue persists, contact Dell Customer Support.",
        "asset.name": "Test_DB_001",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": "MICROSOFT_EXCHANGE_DATABASE",
        "createTime": "2024-12-11T10:07:16.713Z",
        "endTime": "2024-12-11T10:07:39.624Z"
    },
    {
        "category": "CONFIG",
        "classType": "JOB",
        "activityInitiatedType": "MANUAL",
        "result.status": "FAILED",
        "result.error.code": "ASDS0030",
        "result.error.detailedDescription": "Configuration of this protection policy cannot be completed because an error occurred on the application agent host.",
        "result.error.extendedReason": null,
        "result.error.reason": "Unable to configure the policy because of an error on the application agent.",
        "result.error.remediation": "To resolve this issue:\n1. Delete the files in the following directories:\nOn Windows, \\INSTALL_DIR\\AGENTSERVICECONFIG.\nOn Linux and AIX, /opt/dpsapps/agentsvc/config.\n2. In the PowerProtect Data Manager UI, go to Protection > Protection Policies, select this policy, and click Set Lockbox.\nIf the issue persists, contact Dell Customer Support.",
        "asset.name": "Mailbox Database 0597205983",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": "MICROSOFT_EXCHANGE_DATABASE",
        "createTime": "2024-12-11T10:07:16.889Z",
        "endTime": "2024-12-11T10:07:39.865Z"
    },
    {
        "category": "CONFIG",
        "classType": "JOB",
        "activityInitiatedType": "MANUAL",
        "result.status": "FAILED",
        "result.error.code": "ASDS0030",
        "result.error.detailedDescription": "Configuration of this protection policy cannot be completed because an error occurred on the application agent host.",
        "result.error.extendedReason": null,
        "result.error.reason": "Unable to configure the policy because of an error on the application agent.",
        "result.error.remediation": "To resolve this issue:\n1. Delete the files in the following directories:\nOn Windows, \\INSTALL_DIR\\AGENTSERVICECONFIG.\nOn Linux and AIX, /opt/dpsapps/agentsvc/config.\n2. In the PowerProtect Data Manager UI, go to Protection > Protection Policies, select this policy, and click Set Lockbox.\nIf the issue persists, contact Dell Customer Support.",
        "asset.name": "Test_DB_002",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": "MICROSOFT_EXCHANGE_DATABASE",
        "createTime": "2024-12-11T10:07:16.488Z",
        "endTime": "2024-12-11T10:07:40.102Z"
    },
    {
        "category": "PROTECT",
        "classType": "TASK",
        "activityInitiatedType": null,
        "result.status": "UNKNOWN",
        "result.error.code": "ABA0023",
        "result.error.detailedDescription": "Monitoring of this session cannot be completed because of an unknown error occurred during monitoring.",
        "result.error.extendedReason": "",
        "result.error.reason": "Exchange Backups - TEST PLC: Unable to monitor backup/restore session of MICROSOFT_EXCHANGE_DATABASE asset Test_DB_003 in host exchange.demo.local because of an unknown error. The backup/restore session might be running on the client.",
        "result.error.remediation": "Monitor the next few jobs and please ensure network connectivity between agent and PowerProtect Data Manager has no issues and all the services of PowerProtect Data Manager are up and running.",
        "asset.name": "Test_DB_003",
        "asset.type": null,
        "host.name": "exchange.demo.local",
        "host.type": null,
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": null,
        "createTime": "2024-12-11T10:08:08.573Z",
        "endTime": "2024-12-11T10:08:25.434Z"
    },
    {
        "category": "PROTECT",
        "classType": "TASK",
        "activityInitiatedType": null,
        "result.status": "UNKNOWN",
        "result.error.code": "ABA0023",
        "result.error.detailedDescription": "Monitoring of this session cannot be completed because of an unknown error occurred during monitoring.",
        "result.error.extendedReason": "",
        "result.error.reason": "Exchange Backups - TEST PLC: Unable to monitor backup/restore session of MICROSOFT_EXCHANGE_DATABASE asset Mailbox Database 0597205983 in host exchange.demo.local because of an unknown error. The backup/restore session might be running on the client.",
        "result.error.remediation": "Monitor the next few jobs and please ensure network connectivity between agent and PowerProtect Data Manager has no issues and all the services of PowerProtect Data Manager are up and running.",
        "asset.name": "Mailbox Database 0597205983",
        "asset.type": null,
        "host.name": "exchange.demo.local",
        "host.type": null,
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": null,
        "createTime": "2024-12-11T10:08:08.557Z",
        "endTime": "2024-12-11T10:08:25.469Z"
    },
    {
        "category": "PROTECT",
        "classType": "TASK",
        "activityInitiatedType": null,
        "result.status": "UNKNOWN",
        "result.error.code": "ABA0023",
        "result.error.detailedDescription": "Monitoring of this session cannot be completed because of an unknown error occurred during monitoring.",
        "result.error.extendedReason": "",
        "result.error.reason": "Exchange Backups - TEST PLC: Unable to monitor backup/restore session of MICROSOFT_EXCHANGE_DATABASE asset Test_DB_004 in host exchange.demo.local because of an unknown error. The backup/restore session might be running on the client.",
        "result.error.remediation": "Monitor the next few jobs and please ensure network connectivity between agent and PowerProtect Data Manager has no issues and all the services of PowerProtect Data Manager are up and running.",
        "asset.name": "Test_DB_004",
        "asset.type": null,
        "host.name": "exchange.demo.local",
        "host.type": null,
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": null,
        "createTime": "2024-12-11T10:08:08.652Z",
        "endTime": "2024-12-11T10:08:25.479Z"
    },
    {
        "category": "PROTECT",
        "classType": "TASK",
        "activityInitiatedType": null,
        "result.status": "UNKNOWN",
        "result.error.code": "ABA0023",
        "result.error.detailedDescription": "Monitoring of this session cannot be completed because of an unknown error occurred during monitoring.",
        "result.error.extendedReason": "",
        "result.error.reason": "Exchange Backups - TEST PLC: Unable to monitor backup/restore session of MICROSOFT_EXCHANGE_DATABASE asset Test_DB_001 in host exchange.demo.local because of an unknown error. The backup/restore session might be running on the client.",
        "result.error.remediation": "Monitor the next few jobs and please ensure network connectivity between agent and PowerProtect Data Manager has no issues and all the services of PowerProtect Data Manager are up and running.",
        "asset.name": "Test_DB_001",
        "asset.type": null,
        "host.name": "exchange.demo.local",
        "host.type": null,
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": null,
        "createTime": "2024-12-11T10:08:08.860Z",
        "endTime": "2024-12-11T10:08:25.495Z"
    },
    {
        "category": "PROTECT",
        "classType": "TASK",
        "activityInitiatedType": null,
        "result.status": "UNKNOWN",
        "result.error.code": "ABA0023",
        "result.error.detailedDescription": "Monitoring of this session cannot be completed because of an unknown error occurred during monitoring.",
        "result.error.extendedReason": "",
        "result.error.reason": "Exchange Backups - TEST PLC: Unable to monitor backup/restore session of MICROSOFT_EXCHANGE_DATABASE asset Test_DB_002 in host exchange.demo.local because of an unknown error. The backup/restore session might be running on the client.",
        "result.error.remediation": "Monitor the next few jobs and please ensure network connectivity between agent and PowerProtect Data Manager has no issues and all the services of PowerProtect Data Manager are up and running.",
        "asset.name": "Test_DB_002",
        "asset.type": null,
        "host.name": "exchange.demo.local",
        "host.type": null,
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": null,
        "createTime": "2024-12-11T10:08:08.638Z",
        "endTime": "2024-12-11T10:08:25.514Z"
    },
    {
        "category": "PROTECT",
        "classType": "JOB",
        "activityInitiatedType": "MANUAL",
        "result.status": "UNKNOWN",
        "result.error.code": "ABA0023",
        "result.error.detailedDescription": "Monitoring of this session cannot be completed because of an unknown error occurred during monitoring.",
        "result.error.extendedReason": "",
        "result.error.reason": "Exchange Backups - TEST PLC: Unable to monitor backup/restore session of MICROSOFT_EXCHANGE_DATABASE asset Test_DB_002 in host exchange.demo.local because of an unknown error. The backup/restore session might be running on the client.",
        "result.error.remediation": "Monitor the next few jobs and please ensure network connectivity between agent and PowerProtect Data Manager has no issues and all the services of PowerProtect Data Manager are up and running.",
        "asset.name": "Test_DB_002",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": "MICROSOFT_EXCHANGE_DATABASE",
        "createTime": "2024-12-11T10:07:36.259Z",
        "endTime": "2024-12-11T10:08:26.430Z"
    },
    {
        "category": "CONFIG",
        "classType": "TASK",
        "activityInitiatedType": "MANUAL",
        "result.status": "FAILED",
        "result.error.code": "ASDS0030",
        "result.error.detailedDescription": "Configuration of this protection policy cannot be completed because an error occurred on the application agent host.",
        "result.error.extendedReason": null,
        "result.error.reason": "Unable to configure the policy because of an error on the application agent.",
        "result.error.remediation": "To resolve this issue:\n1. Delete the files in the following directories:\nOn Windows, \\INSTALL_DIR\\AGENTSERVICECONFIG.\nOn Linux and AIX, /opt/dpsapps/agentsvc/config.\n2. In the PowerProtect Data Manager UI, go to Protection > Protection Policies, select this policy, and click Set Lockbox.\nIf the issue persists, contact Dell Customer Support.",
        "asset.name": "Mailbox Database 0597205983",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": null,
        "createTime": "2024-12-11T10:07:31.092Z",
        "endTime": "2024-12-11T10:07:37.941Z"
    },
    {
        "category": "CONFIG",
        "classType": "TASK",
        "activityInitiatedType": "MANUAL",
        "result.status": "FAILED",
        "result.error.code": "ASDS0030",
        "result.error.detailedDescription": "Configuration of this protection policy cannot be completed because an error occurred on the application agent host.",
        "result.error.extendedReason": null,
        "result.error.reason": "Unable to configure the policy because of an error on the application agent.",
        "result.error.remediation": "To resolve this issue:\n1. Delete the files in the following directories:\nOn Windows, \\INSTALL_DIR\\AGENTSERVICECONFIG.\nOn Linux and AIX, /opt/dpsapps/agentsvc/config.\n2. In the PowerProtect Data Manager UI, go to Protection > Protection Policies, select this policy, and click Set Lockbox.\nIf the issue persists, contact Dell Customer Support.",
        "asset.name": "Test_DB_003",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": null,
        "createTime": "2024-12-11T10:07:31.258Z",
        "endTime": "2024-12-11T10:07:37.964Z"
    },
    {
        "category": "CONFIG",
        "classType": "TASK",
        "activityInitiatedType": "MANUAL",
        "result.status": "FAILED",
        "result.error.code": "ASDS0030",
        "result.error.detailedDescription": "Configuration of this protection policy cannot be completed because an error occurred on the application agent host.",
        "result.error.extendedReason": null,
        "result.error.reason": "Unable to configure the policy because of an error on the application agent.",
        "result.error.remediation": "To resolve this issue:\n1. Delete the files in the following directories:\nOn Windows, \\INSTALL_DIR\\AGENTSERVICECONFIG.\nOn Linux and AIX, /opt/dpsapps/agentsvc/config.\n2. In the PowerProtect Data Manager UI, go to Protection > Protection Policies, select this policy, and click Set Lockbox.\nIf the issue persists, contact Dell Customer Support.",
        "asset.name": "Test_DB_002",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": null,
        "createTime": "2024-12-11T10:07:30.716Z",
        "endTime": "2024-12-11T10:07:37.874Z"
    },
    {
        "category": "CONFIG",
        "classType": "JOB",
        "activityInitiatedType": "MANUAL",
        "result.status": "FAILED",
        "result.error.code": "ASDS0030",
        "result.error.detailedDescription": "Configuration of this protection policy cannot be completed because an error occurred on the application agent host.",
        "result.error.extendedReason": null,
        "result.error.reason": "Unable to configure the policy because of an error on the application agent.",
        "result.error.remediation": "To resolve this issue:\n1. Delete the files in the following directories:\nOn Windows, \\INSTALL_DIR\\AGENTSERVICECONFIG.\nOn Linux and AIX, /opt/dpsapps/agentsvc/config.\n2. In the PowerProtect Data Manager UI, go to Protection > Protection Policies, select this policy, and click Set Lockbox.\nIf the issue persists, contact Dell Customer Support.",
        "asset.name": "Test_DB_003",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": "MICROSOFT_EXCHANGE_DATABASE",
        "createTime": "2024-12-11T10:07:17.058Z",
        "endTime": "2024-12-11T10:07:39.203Z"
    },
    {
        "category": "CONFIG",
        "classType": "TASK",
        "activityInitiatedType": "MANUAL",
        "result.status": "FAILED",
        "result.error.code": "ASDS0030",
        "result.error.detailedDescription": "Configuration of this protection policy cannot be completed because an error occurred on the application agent host.",
        "result.error.extendedReason": null,
        "result.error.reason": "Unable to configure the policy because of an error on the application agent.",
        "result.error.remediation": "To resolve this issue:\n1. Delete the files in the following directories:\nOn Windows, \\INSTALL_DIR\\AGENTSERVICECONFIG.\nOn Linux and AIX, /opt/dpsapps/agentsvc/config.\n2. In the PowerProtect Data Manager UI, go to Protection > Protection Policies, select this policy, and click Set Lockbox.\nIf the issue persists, contact Dell Customer Support.",
        "asset.name": "Test_DB_001",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": null,
        "createTime": "2024-12-11T10:07:30.952Z",
        "endTime": "2024-12-11T10:07:37.925Z"
    },
    {
        "category": "PROTECT",
        "classType": "JOB",
        "activityInitiatedType": "MANUAL",
        "result.status": "UNKNOWN",
        "result.error.code": "ABA0023",
        "result.error.detailedDescription": "Monitoring of this session cannot be completed because of an unknown error occurred during monitoring.",
        "result.error.extendedReason": "",
        "result.error.reason": "Exchange Backups - TEST PLC: Unable to monitor backup/restore session of MICROSOFT_EXCHANGE_DATABASE asset Test_DB_004 in host exchange.demo.local because of an unknown error. The backup/restore session might be running on the client.",
        "result.error.remediation": "Monitor the next few jobs and please ensure network connectivity between agent and PowerProtect Data Manager has no issues and all the services of PowerProtect Data Manager are up and running.",
        "asset.name": "Test_DB_004",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": "MICROSOFT_EXCHANGE_DATABASE",
        "createTime": "2024-12-11T10:07:35.604Z",
        "endTime": "2024-12-11T10:08:27.420Z"
    },
    {
        "category": "PROTECT",
        "classType": "JOB",
        "activityInitiatedType": "MANUAL",
        "result.status": "UNKNOWN",
        "result.error.code": "ABA0023",
        "result.error.detailedDescription": "Monitoring of this session cannot be completed because of an unknown error occurred during monitoring.",
        "result.error.extendedReason": "",
        "result.error.reason": "Exchange Backups - TEST PLC: Unable to monitor backup/restore session of MICROSOFT_EXCHANGE_DATABASE asset Mailbox Database 0597205983 in host exchange.demo.local because of an unknown error. The backup/restore session might be running on the client.",
        "result.error.remediation": "Monitor the next few jobs and please ensure network connectivity between agent and PowerProtect Data Manager has no issues and all the services of PowerProtect Data Manager are up and running.",
        "asset.name": "Mailbox Database 0597205983",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": "MICROSOFT_EXCHANGE_DATABASE",
        "createTime": "2024-12-11T10:07:36.635Z",
        "endTime": "2024-12-11T10:08:26.936Z"
    },
    {
        "category": "PROTECT",
        "classType": "JOB",
        "activityInitiatedType": "MANUAL",
        "result.status": "UNKNOWN",
        "result.error.code": "ABA0023",
        "result.error.detailedDescription": "Monitoring of this session cannot be completed because of an unknown error occurred during monitoring.",
        "result.error.extendedReason": "",
        "result.error.reason": "Exchange Backups - TEST PLC: Unable to monitor backup/restore session of MICROSOFT_EXCHANGE_DATABASE asset Test_DB_003 in host exchange.demo.local because of an unknown error. The backup/restore session might be running on the client.",
        "result.error.remediation": "Monitor the next few jobs and please ensure network connectivity between agent and PowerProtect Data Manager has no issues and all the services of PowerProtect Data Manager are up and running.",
        "asset.name": "Test_DB_003",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": "MICROSOFT_EXCHANGE_DATABASE",
        "createTime": "2024-12-11T10:07:37.489Z",
        "endTime": "2024-12-11T10:08:27.172Z"
    },
    {
        "category": "PROTECT",
        "classType": "JOB",
        "activityInitiatedType": "MANUAL",
        "result.status": "UNKNOWN",
        "result.error.code": "ABA0023",
        "result.error.detailedDescription": "Monitoring of this session cannot be completed because of an unknown error occurred during monitoring.",
        "result.error.extendedReason": "",
        "result.error.reason": "Exchange Backups - TEST PLC: Unable to monitor backup/restore session of MICROSOFT_EXCHANGE_DATABASE asset Test_DB_001 in host exchange.demo.local because of an unknown error. The backup/restore session might be running on the client.",
        "result.error.remediation": "Monitor the next few jobs and please ensure network connectivity between agent and PowerProtect Data Manager has no issues and all the services of PowerProtect Data Manager are up and running.",
        "asset.name": "Test_DB_001",
        "asset.type": "MICROSOFT_EXCHANGE_DATABASE",
        "host.name": "exchange.demo.local",
        "host.type": "APP_HOST",
        "inventorySource.type": null,
        "protectionPolicy.name": "Exchange Backups - TEST PLC",
        "protectionPolicy.type": "MICROSOFT_EXCHANGE_DATABASE",
        "createTime": "2024-12-11T10:07:37.074Z",
        "endTime": "2024-12-11T10:08:27.781Z"
    }
]